app/state_machines/efile_submission_state_machine.rb

```diff
@@ -84,7 +84,7 @@ class EfileSubmissionStateMachine
       submission.transition_to!(:waiting) if transition.efile_errors.all?(&:auto_wait)
       if transition.efile_errors.all? { |efile_error| EfileError.error_codes_to_retry_once.include?(efile_error.code) }
         already_auto_resubmitted = submission.previously_transmitted_submission && submission.previously_transmitted_submission.efile_submission_transitions.where(to_state: :resubmitted
-          ).any? { |transition| puts(transition.metadata); transition.metadata.dig("auto_resubmitted") }
+          ).any? { |transition| transition.metadata.dig("auto_resubmitted") }
         unless already_auto_resubmitted
           submission.transition_to!(:resubmitted, {auto_resubmitted: true})
```

Comment:
incredibly picky, but im surprised that rubocop doesnt try to enforce spaces around the hash here.

Reply:
@bytheway875 Fascinating! Me too.